app/views/dashboard/index.html.slim

```diff
@@ -52,9 +52,9 @@ h1 Cluster Status
     .panel-heading
       h3.panel-title Nodes
 
-      = link_to kubectl_config_path, id: "download-kubeconfig", class: "btn btn-sm btn-default pull-right", disabled: true do
+      = link_to kubeconfig_path, id: "download-kubeconfig", class: "btn btn-sm btn-default pull-right", disabled: true do
         i.fa.fa-download.fa-fw
-        | kubectl config
+        | kubeconfig
```

Comment:
I wouldn't change this, it's breaking Jenkins (or fix jenkins though).

Reply:
I'll fix automation then.